ssn/rdf/sosa.ttl

```diff
@@ -153,59 +144,52 @@ sosa:Result
   rdfs:label "Result"@en ;
   skos:definition "The Result of an Observation, Actuation, or act of Sampling. To store an observation's simple result value one can use the hasSimpleResult property."@en ;
   skos:example "The value 20 as the height of a certain tree together with the unit, e.g., Meter."@en ;
-
 .
 sosa:Sample
   rdf:type rdfs:Class ;
   rdf:type owl:Class ;
-  rdfs:comment "Feature on which observations may be made, which is intended to be representative of a FeatureOfInterest that is not fully accessible."@en ;
+  rdfs:comment "Feature on which Observations may be made, which is intended to be representative of a FeatureOfInterest that is not fully accessible."@en ;
   rdfs:comment "Physical samples are sometimes known as 'specimens'."@en ;
   rdfs:comment """Samples are artifacts of an observational strategy, and have no significant function outside of their role in the observation process. The characteristics of the samples themselves are of little interest, except perhaps to the manager of a sampling campaign.
 
 A Sample is intended to sample some FatureOfInterest in an application domain, so there is an expectation of at least one sampledFeature property. However, in some cases the identity, and even the exact type, of the sampled feature may not be known when observations are made using the sampling features."""@en ;
   rdfs:label "Sample"@en ;
-  skos:definition "Feature on which observations may be made, which is intended to be representative of a FeatureOfInterest that is not fully accessible."@en ;
+  skos:definition "Feature on which Observations may be made, which is intended to be representative of a FeatureOfInterest that is not fully accessible."@en ;
```

Comment:
I would only us the upper case for the class name, not for a collection. I.e., for instance, 'Feature on which an Observation is made' versus 'Feature on which observations are made'. This is how we did it in the entire file before.